pkg/cluster/task/init_config.go

```diff
@@ -45,6 +46,9 @@ func (c *InitConfig) Execute(ctx *Context) error {
 
 	err := c.instance.InitConfig(exec, c.clusterName, c.clusterVersion, c.deployUser, c.paths)
 	if err != nil {
+		if errors.Cause(err) == meta.ErrorCheckConfig {
```

Comment:
`if c.ignoreCheck && errors.Cause(err) == meta.ErrorCheckConfig`.
Please add some unit tests for this. It's better to mock a `meta.Instance` to test the `InitConfig` task.